src/components/ResponsiveSidebar.js

```diff
@@ -0,0 +1,149 @@
+import {
+  Box,
+  Drawer,
+  Divider,
+  Typography,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Stack,
+} from '@mui/material';
+import {
+  HomeOutlined,
+  AccountBalanceWalletOutlined,
+  AccountCircleOutlined,
+  ContactSupportOutlined,
+  PowerSettingsNewOutlined,
+} from '@mui/icons-material';
+
+export const drawerWidth = 241;
+
+function ResponsiveSidebar() {
+  const drawer = (
+    <div>
+      <Stack
+        sx={{
+          backgroundColor: '#014656',
+          color: '#ffff',
+          direction: 'column',
+          height: '80px',
+          width: '100%',
+        }}
+      >
+        <Typography
+          variant="caption"
+          sx={{ textAlign: 'right', marginRight: '7px' }}
+        >
+          Viasat
+        </Typography>
+        <Typography
+          variant="subtitle1"
+          sx={{ fontSize: '12px', marginLeft: '7px' }}
+        >
+          Alex Garcia
+        </Typography>
+        <Typography
+          variant="caption"
+          sx={{ fontSize: '12px', opacity: 0.8, marginLeft: '7px' }}
+        >
+          alex.garcia@gmail.com
+        </Typography>
+      </Stack>
+      <List>
+        <ListItem>
+          <ListItemIcon>
+            <HomeOutlined />
+          </ListItemIcon>
+          <ListItemText>HOME</ListItemText>
+        </ListItem>
+        <ListItem
+          button
+          sx={{
+            backgroundColor: '#F2F5F8',
+            color: '#04819E',
+            boxShadow:
+              ' inset 0px -1px 0px rgba(0, 0, 0, 0.15), inset 3px 0px 0px #04819E',
+          }}
+        >
+          <ListItemIcon>
+            <AccountBalanceWalletOutlined sx={{ color: '#04819E' }} />
+          </ListItemIcon>
+          <ListItemText primary="BILLING" />
+        </ListItem>
+        <ListItem>
+          <ListItemIcon>
+            <AccountCircleOutlined />
+          </ListItemIcon>
+          <ListItemText primary="ACCOUNT SETTINGS" />
+        </ListItem>
+        <ListItem>
+          <ListItemIcon>
+            <ContactSupportOutlined />
+          </ListItemIcon>
+          <ListItemText primary="HELP FAQ" />
+        </ListItem>
+      </List>
```

Comment:
There is a common structure between these list items, they each have a title and an icon. 

As an improvement, I suggest you try to build an array of objects upon which to map out these `ListItem`s. Such an object could have a `title` and `icon` property, as well as an additional property holding a boolean variable, that can be used to apply the extra styles and functionality for the active list item.